.vscode/settings.json

```diff
@@ -16,7 +16,6 @@
       "source.organizeImports": "explicit"
     }
   },
-
```

Comment:
Was this done by the json autoformatting?

Reply:
Good catch, thanks. 17bdae38a313c5f135079f440f813b9d8934be03.